### PR TITLE
Upload incident type + field fix

### PR DESCRIPTION
**Pull request by orlichter1**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: No issue yet

## Description
Incident types and fields didn't upload correctly. They now upload.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by jochman — CHANGES_REQUESTED** [review]
Look fine.
add tests and comments on why you did so.

**Review by ShahafBenYakir — COMMENTED** [review]
Add tests and changlog

**Comment by orlichter1** [comment]
@ShahafBenYakir this is a fix to something I merged yesterday wasn't released yet so I'm not adding new changelogs (as it is included in yesterday's)

File: demisto_sdk/commands/upload/tests/uploader_test.py [hunk 1]
```diff
@@ -217,6 +230,64 @@ def test_upload_incident_field_positive(demisto_client_configure, mocker):
...
+    incident_field_name = "XDR_Alert_Count.json"
+    incident_field_path = f"{git_path()}/demisto_sdk/tests/test_files/CortexXDR/IncidentFields/{incident_field_name}"
+    uploader = Uploader(input=incident_field_path, insecure=False, verbose=False)
+    uploader.client.import_incident_fields = MagicMock(side_effect=save_file)
+    uploader.upload()
+
+    with open(incident_field_path) as json_file:
+        incident_field_data = json.load(json_file)
+
+    assert json.loads(DATA)['incidentFields'][0] == incident_field_data
```
Comment (jochman) [inline demisto_sdk/commands/upload/tests/uploader_test.py:288]: Delete the file after the test. why dont use tempfile?

**Review by jochman — CHANGES_REQUESTED** [review]

File: demisto_sdk/commands/upload/tests/uploader_test.py (hunk 1 above)
Comment (orlichter1) [reply demisto_sdk/commands/upload/tests/uploader_test.py:288]: The file is deleted. DATA is a string extracted from the deleted temporary file. (That's where I had issues in the unittests)

**Review by jochman — APPROVED** [review]

**Pull request merged**